hollow-diff-ui/src/main/java/com/netflix/hollow/diff/ui/java/OptionalDependencyHelper.java

```diff
@@ -14,16 +14,16 @@
  *     limitations under the License.
  *
  */
-package com.netflix.hollow.diff.ui.jetty;
+package com.netflix.hollow.diff.ui.java;
 
-import com.netflix.hollow.ui.jetty.AbstractOptionalDependencyHelper;
+import com.netflix.hollow.ui.java.AbstractOptionalDependencyHelper;
 
 final class OptionalDependencyHelper extends AbstractOptionalDependencyHelper {
```

Comment:
we can drop OptionalDependencyHelper now since the new http server dependencies will be "api" dependencies for hollow tools

Reply:
done